### DISCUSSION TITLE
Is there a difference between android-5 and android-7 other than the bootstrtap?

### DISCUSSION THREAD

**Discussion by MatanZ** [termux/termux-app#2989] [Q&A] [answered]
I build apks without the bootstrap images, and install them on different phones with version 5 and 7. Do I need to compile two different apks, or are the packages the same?

**Comment by agnostic-apollo** [comment, marked as answer]
Termux app behaviour depends on the bootstrap variant in the apk. Do not install android-5 variant on Android 7+. The packages are different too.

https://github.com/termux/termux-app/blob/211340781bcb0576ffe100fda2e0ed87d7333d53/app/build.gradle#L6-L12
